<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>1019</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>1021</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>接收区</string>
    </property>
    <widget class="QLabel" name="mStateLabel_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="rcvTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>991</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>991</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="rcvCleanButton">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rcvFileNameLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="rcvFileSelectButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rcvSaveButton">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rcvToFileCheckBox">
        <property name="text">
         <string>输出到文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rcvHexCheckBox">
        <property name="text">
         <string>十六进制</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>1021</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>发送区</string>
    </property>
    <widget class="QCheckBox" name="sndHexCheckBox">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制发送</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="sndTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>991</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sndSendButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sndCRCheckBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>\r</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sndNewLineCheckBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>\n</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mTimeLineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>39</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sndNewLineCheckBox_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>自动发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="rcvSysTimeCheckBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>300</y>
      <width>109</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>系统时间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="SerialPortWidget" name="mSerialPortWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>891</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SerialPortWidget</class>
   <extends>QWidget</extends>
   <header>SerialPortWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
